### DISCUSSION TITLE
Cant sign upo with github despite correct username and password - resolved

### DISCUSSION THREAD

**Discussion by chrishj59** [vercel/vercel#5214] [Help] [answered]
Vercel appears to require sign-up using a GitHub account. I cannot log in to GitHub using the Vercel log in page despite using the same user name and password as when I log in to GitHub directly.
I notice that I do not get the 2-factor SMS  message when attempting login via Vercel

**Comment by mcsdevv** [comment, marked as answer]
Hello Chris,

I'm sorry to hear that, can you please submit a [support request](https://vercel.com/support/request) so that we can investigate further?

**Comment by yugasun** [comment]
Same issue for me o(╯□╰)o

**Comment by adnabb** [comment]
I have the same problem. Can you login in well now?

**Comment by mcsdevv** [comment]
Hello @yugasun, @adnabb, if you're having difficulty with sign up, please get in touch as mentioned above.

**Comment by yugasun** [comment]
@mcsdevv Yeah, I have send the question email, and waiting for reply.

**Comment by chrishj59** [comment]
I resolved this by marking Vercel as an allowed 3rd party on my GitHUb account. After this the login worked.
Perhaps the procedure note should be updated to include tgis step
